src/src/rx_main.cpp

```diff
@@ -936,6 +936,9 @@ bool ICACHE_RAM_ATTR ProcessRFPacket(SX12xxDriverCommon::rx_status const status)
 
 bool ICACHE_RAM_ATTR RXdoneISR(SX12xxDriverCommon::rx_status const status)
 {
+    if (LQCalc.currentIsSet() && connectionState == connected)
```

Comment:
Minor annoyance, that causes major frustrations when debugging...
Can we add some parentheses to these one line `if` statements?